tools/external-grading-load-test/main.js

```diff
@@ -96,8 +96,7 @@ async.series([
                 return next(null);
             });
             request.on('error', (err) => {
-                logger.error(`Error submitting job ${n}!`);
-                logger.error(err);
+                logger.error(`Error submitting job ${n}!`, err);
```

Comment:
```suggestion
                logger.error(`Error submitting job ${n}`, err);
```
just for consistency -- feel free to disregard though

Reply:
Oh definitely, I'm not a fan of exclamation marks. Thanks!